docs/index.mdx

```diff
@@ -23,30 +21,32 @@ Some of the main Next.js features include:
 | [Styling](/docs/app/building-your-application/styling)                   | Support for your preferred styling methods, including CSS Modules, Tailwind CSS, and CSS-in-JS                                                                                                   |
 | [Optimizations](/docs/app/building-your-application/optimizing)          | Image, Fonts, and Script Optimizations to improve your application's Core Web Vitals and User Experience.                                                                                        |
 | [TypeScript](/docs/app/building-your-application/configuring/typescript) | Improved support for TypeScript, with better type checking and more efficient compilation, as well as custom TypeScript Plugin and type checker.                                                 |
-| [API Reference](/docs/app/api-reference)                                 | Updates to the API design throughout Next.js. Please refer to the API Reference Section for new APIs.                                                                                            |
 
 ## How to Use These Docs
 
-The sections and pages of the docs are organized sequentially, from basic to advanced, so you can follow them step-by-step when building your Next.js application. However, you can read them in any order or skip to the pages that apply to your use case.
+On the left side of the screen, you'll find the docs navbar. The pages of the docs are organized sequentially, from basic to advanced, so you can follow them step-by-step when building your application. However, you can read them in any order or skip to the pages that apply to your use case.
 
-At the top of the sidebar, you'll notice a dropdown menu that allows you to switch between the **App Router** and the **Pages Router** features. Since there are features that are unique to each directory, it's important to keep track of which tab is selected.
+On the right side of the screen, you'll see a table of contents that makes it easier to navigate between sections of a page. If you need to quickly find a page, you can use the search bar at the top, or the search shortcut (`Ctrl+K` or `Cmd+K`).
 
-On the right side of the page, you'll see a table of contents that makes it easier to navigate between sections of a page. The breadcrumbs at the top of the page will also indicate whether you're viewing App Router docs or Pages Router docs.
+To get started, checkout the [Installation](/docs/getting-started/installation) guide. If you're new to React, we recommend reading the [React Essentials](/docs/getting-started/react-essentials) page.
 
-To get started, checkout the [Installation](/docs/getting-started/installation). If you're new to React or Server Components, we recommend reading the [React Essentials](/docs/getting-started/react-essentials) page.
+## App Router vs Pages Router
```

Comment:
The content of this section mainly talks about the switcher, which is great! But the title of the section points towards more of a comparison of routers. Maybe we can change the title to `### Switching between router docs` and make it a child of the above section (an H3 as opposed to an H2)